opentelemetry-exporters-newrelic-auto/src/main/java/com/newrelic/telemetry/opentelemetry/export/auto/NewRelicConfiguration.java

```diff
@@ -30,4 +30,8 @@ static boolean shouldEnableAuditLogging(Config config) {
   static String getServiceName(Config config) {
     return config.getString(NEW_RELIC_SERVICE_NAME, DEFAULT_NEW_RELIC_SERVICE_NAME);
   }
+
+  static boolean isNotBlank(String s) {
```

Comment:
I am not stoked on this name.  The adjective "blank" to me suggests  a string containing only whitespace...which this isn't.  How about `isNotEmpty()` or `hasContent` as alternatives?

Reply:
I don't love any of them. It's really `isUseful()`.  :).

Reply:
Since it's in the config class, how about `isSpecified()` ?

Reply:
This makes me think we should wrap the Config in our own class that provides a more readable interface. That might be a bit beyond the scope of this PR, though.

Reply:
That's legit, and I think `isSpecified()` is still an improvement over `isNotBlank()` :). Thanks!